src/site/_includes/js/eligibility.js

```diff
@@ -258,6 +258,22 @@ const cnst = {
     // https://www.ecfr.gov/current/title-7/subtitle-B/chapter-II/subchapter-A/part-246#p-246.2(Children)
     CHILD_EXIT_AGE: 5, // Birthday at which a child is ineligible.
   },
+  clipper: {
```

Comment:
Can you keep the keys of the `cnst` object in alphabetical order?  i.e. move `clipper` up?

Reply:
Done in [f3038e7](https://github.com/theunitedeffort/theunitedeffort.org/pull/610/commits/f3038e71984f7e2cbbb53b7d59b5ede13d9315f8)